charts/stable/mealie/values.yaml

```diff
@@ -1,34 +1,28 @@
-#
 # IMPORTANT NOTE
 #
 # This chart inherits from our common library chart. You can check the default values/options here:
 # https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
 #
-
+---
 image:
   # -- image repository
   repository: hkotel/mealie
   # -- image tag
   # @default -- chart.appVersion
-  tag:
+  tag: frontend-v1.0.0beta-2
```

Comment:
The `appVersion` field in Chart.yaml refers to version v1.0.0-beta-1, should they not be in sync?

Reply:
Since there is a tag prefix we can't use chart.appVersion by default. I have made app version match the suffix though.